src/RootSelector.js

```diff
@@ -0,0 +1,7 @@
+// import fetchRecipe from './Recepies/SearchBar/selectors';
```

Comment:
What purpose does this file serve?

Reply:
This file has no purpose within the application for now and I have removed it.